projects/components/src/filtering/filter-bar/filter-chip/filter-chip.component.ts

```diff
@@ -120,8 +120,7 @@ export class FilterChipComponent implements OnInit, OnChanges {
   private mapToComboBoxOption(filter: IncompleteFilter): ComboBoxOption<IncompleteFilter> {
     return {
       text: filter.userString,
-      value: filter,
-      tooltip: `${filter.userString} (${filter.field})`
+      value: filter
```

Comment:
How are tooltips now handled? Or at least, how do we let users know the underlying field backing the display name?

Reply:
I actually removed them, this was one of the things I wanted to ask. In the existing UI, they're:
- The underlying field isn't actually used for matching a user's string that I could tell
- hard to find, since you have to hover on the popup
- very strange to read, since this key is thrown after the user string

Example from existing ui: 
![image](https://user-images.githubusercontent.com/45047841/149206786-21bce7c7-65b4-41a0-bc3b-0c6f6dbb0c63.png)
![image](https://user-images.githubusercontent.com/45047841/149206851-987cc8e5-40f8-4db7-9129-1c5bae3c8896.png)

Reply:
Fair points. Knowing the underlying keys was an ask early on, so we've always had the tooltip available. Not sure if users are even asking for that info. If they are, I'm surprised we haven't gotten feedback about the discoverability of it as you've noted.

I'm fine removing them as long as its a quick fix to add back if an issue arrises.

Reply:
Yeah I remember the ask, but not sure why it was there given that we don't expose keys to users anywhere else. Easy enough to restore if the requirement comes back - hopefully can better understand the ask at that point.